plugin/pxr/maya/lib/usdMaya/translatorModelAssembly.cpp

```diff
@@ -129,22 +129,19 @@ _AddReference(
     std::string refAssetPath,
     SdfPath refPrimPath = SdfPath())
 {
-    if (prim.HasAuthoredReferences()){
```

Comment:
Removing this wont break us in some other place?

Reply:
Nah, it was me putting this in, and was not properly adding references that were not overrides due to this. So removing now.